lib/index.js

```diff
@@ -273,6 +273,9 @@ function makeEventError(error) {
 function extractErrors(obj) {
   var errors = [];
 
+  if (obj instanceof Error) {
+    errors.push(obj);
```

Comment:
Ah, I guess if an Error object had nested Errors on its keys we might only want the top one.